Sources/Host/Darwin/Platform.cpp

```diff
@@ -10,21 +10,74 @@
 
 #include "DebugServer2/Host/Platform.h"
 #include "DebugServer2/Host/Darwin/LibProc.h"
+#include "DebugServer2/Utils/Log.h"
 
 #include <sys/utsname.h>
 
 #include <algorithm>
 #include <cctype>
 #include <cstdlib>
 #include <cstring>
+#include <fstream>
+#include <iostream>
+#include <string>
 
 namespace ds2 {
 namespace Host {
 namespace Darwin {
 
 char const *Platform::GetOSTypeName() { return "macosx"; }
 
-char const *Platform::GetOSVendorName() { return "unknown"; }
+char const *Platform::GetOSVersion() {
+  static char version[9] = {0};
+
+  /*
+   * Mac doesn't offer a C/C++ call to get the OS version. SystemVersion.plist
+   * is use buy officiel application to get the version, so I think it's ok
+   * to read it. An xml parser will be nice, but it will also add a lot of
+   * deps to read one static value.
+   *
+   * TODO: Found a better way to do that
```

Comment:
I think you're right, and there probably isn't a better way to do that, so you can remove this TODO.